internal/worker/worker.go

```diff
@@ -39,3 +39,5 @@ func (w *Worker) Run() {
 		}
 	}
 }
+
```

Comment:
maybe we should delete this as it suggests 😄

Reply:
Yeah I will remove it after CI will build images :smile: